docs/android/faq.md

```diff
@@ -89,6 +89,20 @@ Newer AGP versions provide a more performant API for bytecode instrumentation am
 
 We have benchmarked the Embrace SDK's performance during app launch between 10 and 50 milliseconds. In practice, this is between 1-3% of typical app launch time.
 
+### **What is the impact of adding Embrace Android SDK regarding app performance, device battery/data usage/disk space?**
+
+TL;DR _The impact depends on how you use the API._
+
+The incremental battery usage is negligible because we are mostly dormant unless telemetry is being recorded, in which case the CPU should already be active. We do have a thread that monitors the main thread for ANRs if that feature is enabled when the app is in the foreground, but that only pings the thread every 100ms
```

Comment:
```suggestion
The incremental battery usage is negligible because we are mostly dormant unless telemetry is being recorded, in which case the CPU should already be active. We do have a thread that monitors the main thread for ANRs if that feature is enabled when the app is in the foreground, but that only pings the thread every 100ms.
```